docs/docs/renaming-table.md

```diff
@@ -4,3 +4,36 @@ title: renaming-table
 ---
 
 Renaming a table may break existing clients.
+
+## Solutions for renaming tables
+
+### Don't rename the table
+
+This is the simplest solution. If you're using an ORM (Object Relational Mapper), you can rename the object in your code, but leave the SQL table name as is.
+
+### Rename with minor locking via database views
```

Comment:
not sure, but I thought there was some caveat with foreign keys and oids

Reply:
doesn't seem like it